JShellWrapper/src/test/java/JShellWrapperTest.java

```diff
@@ -67,12 +65,10 @@ void testExpressionResult() {
 
                 false
                 """);
-        evalTest(
-                """
-            eval
-            1
-            2+2""",
-                """
+        evalTest("""
+                eval
+                1
+                2+2""", """
```

Comment:
Did you check that the tests still work ?

Reply:
as i said on [the other comment](https://github.com/Together-Java/JShellPlaygroundBackend/pull/50#discussion_r1769172956)

Reply:
Same